src/masternodes/changiintermediates.h

```diff
@@ -0,0 +1,23 @@
+#ifndef DEFI_MASTERNODES_CHANGIINTERMEDIATES_H
+#define DEFI_MASTERNODES_CHANGIINTERMEDIATES_H
+
+#include <consensus/params.h>
+#include <consensus/tx_check.h>
+#include <masternodes/evm.h>
+#include <masternodes/masternodes.h>
+#include <masternodes/mn_checks.h>
+
+/// The file and class that has all the old things that will be removed.
+/// The goal of this entire class and file is to be removed. It's a collection of the 
+/// accumulated bugs in early releases. 
+class ChangiBuggyIntermediates {
```

Comment:
We only need a function, this class is superfluous.

Reply:
There are other items that need lifting for branching?

Reply:
Any of those branches need to be inside a class?

Reply:
The idea is to refactor them to them so all the buggy code can be brought into one place for removal. And forces us to cleanup the code paths prior to removing the forks.